coordinator/src/services/translation/cancellation/cancellation-response.ts

```diff
@@ -50,18 +51,41 @@ function createBundleEntries(cancellationResponse: CancellationResponse) {
     fhirPractitionerRole: fhirResponsiblePartyPractitionerRole
   } = convertAgentPerson(cancellationResponse.responsibleParty.AgentPerson)
 
-  const {
-    fhirPractitioner: fhirAuthorPractitioner,
-    fhirLocations: fhirAuthorLocations,
-    fhirHealthcareService: fhirAuthorHealthcareService,
-    fhirPractitionerRole: fhirAuthorPractitionerRole
-  } = convertAgentPerson(cancellationResponse.author.AgentPerson)
+  const bundleResources: Array<fhir.Resource> = [
+    fhirPatient,
+    fhirResponsiblePartyPractitioner,
+    ...fhirResponsiblePartyLocations,
+    fhirResponsiblePartyHealthcareService,
+    fhirResponsiblePartyPractitionerRole
+  ]
+
+  let requesterId = fhirResponsiblePartyPractitioner.id
+
+  if (!isDeepStrictEqual(
+    cancellationResponse.responsibleParty.AgentPerson,
```

Comment:
Would this look neater if `cancellationResponse.responsibleParty.AgentPerson` and `cancellationResponse.author.AgentPerson` were pulled out into a variable? They could then be used in the performer checks too.